surveys/surveys.go

```diff
@@ -54,134 +60,140 @@ func LauncherSchemaFromFilename(filename string) LauncherSchema {
-		LauncherSchemaFromFilename("test_radio_mandatory_with_optional_other.json"),
-		LauncherSchemaFromFilename("test_radio_mandatory_with_overridden_error.json"),
-		LauncherSchemaFromFilename("test_radio_optional.json"),
-		LauncherSchemaFromFilename("test_radio_optional_with_mandatory_other.json"),
-		LauncherSchemaFromFilename("test_radio_optional_with_mandatory_other_overridden_error.json"),
-		LauncherSchemaFromFilename("test_radio_optional_with_optional_other.json"),
-		LauncherSchemaFromFilename("test_relationship_household.json"),
-		LauncherSchemaFromFilename("test_repeating_and_conditional_routing.json"),
-		LauncherSchemaFromFilename("test_repeating_household.json"),
-		LauncherSchemaFromFilename("test_repeating_household_routing.json"),
-		LauncherSchemaFromFilename("test_routing_date_equals.json"),
-		LauncherSchemaFromFilename("test_routing_date_greater_than.json"),
-		LauncherSchemaFromFilename("test_routing_date_less_than.json"),
-		LauncherSchemaFromFilename("test_routing_date_not_equals.json"),
-		LauncherSchemaFromFilename("test_routing_group.json"),
-		LauncherSchemaFromFilename("test_routing_number_equals.json"),
-		LauncherSchemaFromFilename("test_routing_number_greater_than.json"),
-		LauncherSchemaFromFilename("test_routing_number_greater_than_or_equal.json"),
-		LauncherSchemaFromFilename("test_routing_number_less_than.json"),
-		LauncherSchemaFromFilename("test_routing_number_less_than_or_equal.json"),
-		LauncherSchemaFromFilename("test_routing_number_not_equals.json"),
-		LauncherSchemaFromFilename("test_routing_on_multiple_select.json"),
-		LauncherSchemaFromFilename("test_skip_condition.json"),
-		LauncherSchemaFromFilename("test_skip_condition_block.json"),
-		LauncherSchemaFromFilename("test_skip_condition_group.json"),
-		LauncherSchemaFromFilename("test_summary.json"),
-		LauncherSchemaFromFilename("test_section_summary.json"),
-		LauncherSchemaFromFilename("test_sum_equal_validation_against_total.json"),
-		LauncherSchemaFromFilename("test_sum_equal_or_less_validation_against_total.json"),
-		LauncherSchemaFromFilename("test_sum_less_validation_against_total.json"),
-		LauncherSchemaFromFilename("test_sum_multi_validation_against_total.json"),
-		LauncherSchemaFromFilename("test_titles.json"),
-		LauncherSchemaFromFilename("test_titles_radio_and_checkbox.json"),
-		LauncherSchemaFromFilename("test_titles_within_repeating_blocks"),
-		LauncherSchemaFromFilename("test_view_submitted_response.json"),
-		LauncherSchemaFromFilename("test_textarea.json"),
-		LauncherSchemaFromFilename("test_textfield.json"),
-		LauncherSchemaFromFilename("test_timeout.json"),
-		LauncherSchemaFromFilename("test_total_breakdown.json"),
-		LauncherSchemaFromFilename("test_unit_patterns.json"),
+func GetAvailableSchemas() LauncherSchemas {
+	schemaList := LauncherSchemas{
+		Live: []LauncherSchema{
+			LauncherSchemaFromFilename("0_star_wars.json"),
```

Comment:
Can we go through as say what is actually live before merging. For example Star Wars isn't surely.

Reply:
The `Live` and `Test` categories are a bit confusing? we could have a schema that isnt a test schema, but isn't `Live` as the business would call it.
But Star Wars should be in the other list

Reply:
I thought we always wanted 3 categories, Business/Social/Test